lessons/Data-Science/histograms-visualize/langs/en-us/pages/making-histograms.adoc

```diff
@@ -0,0 +1,60 @@
+= Making Histograms
+
+== By Hand
+
+Suppose we have a dataset for a group of 50 adults, showing the number of teeth each person has...
+
+@vspace{1ex}
+
+@indented{
+@big{
+0, 0, 0, 0, 0, 22, 26, 27, 28, 28, 28, 28, 29, 29, 29, 30, 30, 30, 30, 30, 31, 31, 31, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32, 32
+}
+}
+
+@n Use the data to complete the frequency table below. (One row has been completed for you.)
+
+@vspace{1ex}
+
+[cols="2a,^1a,^1a,^1a,^1a,^1a,^1a,^1a", options= "header", stripes ="none"]
+|===
+| number of teeth 	| 0-4 		| 5-9 		| 10-14 	| 15-19 	| 20-24		| 25-29  	|30-34
+| frequency			| @ifsoln{4}| @ifsoln{0}| @ifsoln{0}| @ifsoln{0}| @ifsoln{1}| @ifsoln{9}| 35
+|===
+
+@n Use the frequency table to draw a histogram below, filling in each interval so that its height is equal to the frequency.
+
+@vspace{1ex}
+
+@ifnotsoln{ @center{@image{../images/teeth-histogram.png, 400}}}
+@ifsoln{ @center{@image{../images/teeth-histogram-soln.png, 400}}}
+
+== In @proglang
+
+[.linkInstructions]
+Open the @starter-file{tooth-data} copy@ifproglang{pyret}{, and click "Run".}
+
+@n Type `tooth-table` in the Interactions window. Press enter. What do you see? @fitb{}{@ifsoln{The tooth data from above, with names!}}
+
+@n Type `count(tooth-table, "num-teeth")` in the Interactions window and press enter. How is the frequency table created in Pyret different from the one that you created, above? @fitb{}{@ifsoln{Different intervals were used.}}
+
+@fitb{}{}
+
+@n What bin sized was used (above) for the Tooth Data frequency table and the histogram? @fitb{5em}{@ifsoln{5}}
```

Comment:
@retabak We haven't introduced the word bin before... 

After wondering whether we should:

- word the question without the word bin
- introduce the word bin on the page
- teach about bin in advance of this in the lesson

I think we're back to the need for a live survey to kick this lesson off. And some text in the lesson to introduce bin in response to the live survey...

Reply:
@retabak 
ok to wait. added a note about it to the live survey issue but I don't personally find the word bin intuitive at all